views/helpers/index.js

```diff
@@ -15,18 +15,22 @@
 
 'use strict';
 const moment = require('moment');
+const parseColor = require('parse-color');
 
 exports.contrastColor = function(hexcolor) {
   if (!hexcolor) {
     return 'white';
   }
 
-  if (hexcolor[0] === '#') {
-    hexcolor = hexcolor.substr(1, hexcolor.length);
+  // Assume that a 6 digit string is a color.
+  if (hexcolor.length === 6 && hexcolor[0] !== '#') {
+    hexcolor = '#' + hexcolor;
   }
-  const r = parseInt(hexcolor.substr(0, 2), 16);
-  const g = parseInt(hexcolor.substr(2, 2), 16);
-  const b = parseInt(hexcolor.substr(4, 2), 16);
+
+  const parsedColor = parseColor(hexcolor);
```

Comment:
L30-L33 could also be written as

```js
const [r, g, b] = parseColor(hexcolor).rgb;
```

via [destructuring assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Basic_variable_assignment).